### ISSUE TITLE
Use findDOMNode from react-dom

### ISSUE THREAD

**Issue by rkovacevic** [gaearon/redux-devtools#195]
In LogMonitor, it's logging a warning:

Warning: React.findDOMNode is deprecated. Please use ReactDOM.findDOMNode from require('react-dom') instead.

**Comment by veggiemonk** [comment]
My guess:

> React 0.14 support is coming in the next branch.
Helps us test it to get it sooner.

**Comment by rkovacevic** [comment]
I am indeed on 0.14, and I'm helping test it :)

Forgot to mention that.

It doesn't seem fixed in 'next', though:

https://github.com/gaearon/redux-devtools/blob/0a1458cdbd50ea2a345fb8d913f7e0d1c763f42b/src/react/LogMonitor.js#L1

On Sat, Nov 28, 2015 at 5:17 PM, Julien Bisconti <notifications@github.com>
wrote:

> My guess:
>
> React 0.14 support is coming in the next branch.
> Helps us test it to get it sooner.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gaearon/redux-devtools/issues/195#issuecomment-160315617>
> .
>



-- 
Robert Kovačević

E-mail: robert.kovacevic1@gmail.com
Tel: +385 91 602 7569
Skype: robert.kovacevic1
LinkedIn: http://hr.linkedin.com/in/rkovacevic

**Comment by veggiemonk** [comment]
Other guess:

https://github.com/gaearon/redux-devtools-log-monitor

**Comment by ellbee** [comment]
Which version of Redux Devtools are you using?
 
In the `next` branch (v.3.0.0-beta3) the log monitor has been removed and put into its [own repo](https://github.com/gaearon/redux-devtools-log-monitor). I am using the [`next` branch](https://github.com/gaearon/redux-devtools/releases/tag/v3.0.0-beta-3) with the extracted log monitor and it is working well for me. I found the examples in the `next` branch to be super helpful in figuring out how to get set up.
